src/components/Dashboard/views/UpgradeToPro/UpgradeToPro.tsx

```diff
@@ -8,7 +8,7 @@ import Check from '@material-ui/icons/Check';
 import GridItem from '../../Grid/GridItem';
 import GridContainer from '../../Grid/GridContainer';
 import { Danger } from '../../../common/Typography/Danger';
-import Success from '../../Typography/Success';
+import { Success } from '../../../common/index';
```

Comment:
no need to specify the index file and this can be merged with card in one import statement

Reply:
this is no longer relevant - these commit have been clobbered